github/resource_github_actions_runner_group_test.go

```diff
@@ -26,6 +26,8 @@ func TestAccGithubActionsRunnerGroup(t *testing.T) {
 			resource "github_actions_runner_group" "test" {
 			  name       = github_repository.test.name
 			  visibility = "all"
+				restricted_to_workflows = true
```

Comment:
i don't think i can run these tests... i think they require an enterprise account?

Reply:
On main, I'm getting a failure on these acceptance tests:

```
--- FAIL: TestAccGithubActionsRunnerGroup (47.67s)
    --- FAIL: TestAccGithubActionsRunnerGroup/creates_runner_groups_without_error (12.52s)
        --- SKIP: TestAccGithubActionsRunnerGroup/creates_runner_groups_without_error/with_an_anonymous_account (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/creates_runner_groups_without_error/with_an_individual_account (0.00s)
        --- FAIL: TestAccGithubActionsRunnerGroup/creates_runner_groups_without_error/with_an_organization_account (12.52s)
    --- FAIL: TestAccGithubActionsRunnerGroup/manages_runner_visibility (11.71s)
        --- SKIP: TestAccGithubActionsRunnerGroup/manages_runner_visibility/with_an_anonymous_account (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/manages_runner_visibility/with_an_individual_account (0.00s)
        --- FAIL: TestAccGithubActionsRunnerGroup/manages_runner_visibility/with_an_organization_account (11.71s)
    --- FAIL: TestAccGithubActionsRunnerGroup/imports_an_all_runner_group_without_error (11.47s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_an_all_runner_group_without_error/with_an_anonymous_account (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_an_all_runner_group_without_error/with_an_individual_account (0.00s)
        --- FAIL: TestAccGithubActionsRunnerGroup/imports_an_all_runner_group_without_error/with_an_organization_account (11.47s)
    --- PASS: TestAccGithubActionsRunnerGroup/imports_a_private_runner_group_without_error (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_a_private_runner_group_without_error/with_an_anonymous_account (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_a_private_runner_group_without_error/with_an_individual_account (0.00s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_a_private_runner_group_without_error/with_an_organization_account (0.00s)
    --- FAIL: TestAccGithubActionsRunnerGroup/imports_a_selected_runner_group_without_error (11.98s)
        --- SKIP: TestAccGithubActionsRunnerGroup/imports_a_selected_runner_group_without_error/with_an_individual_account (0.00s)
        --- FAIL: TestAccGithubActionsRunnerGroup/imports_a_selected_runner_group_without_error/with_an_organization_account (11.98s)
FAIL
```

On this branch, I'm seeing a panic on line 179 of github/resource_github_actions_runner_group.go:

`d.Set("selected_workflows", runnerGroup.SelectedWorkflows)`. 

The error reads `panic: runtime error: invalid memory address or nil pointer dereference`. Can you reproduce that when you run these tests on the branch locally?

Reply:
I'm not able to successfully run the `TestAccGithubActionsRunnerGroup` tests even from the `main` branch, so it makes it difficult for me to verify my own PR (other tests work file however). Hm...

Reply:
Ohh that makes sense. We currently don't have a framework in place for enterprise testing, although it's on the TODO list in the effort starting with #1414.